xline/src/server/kv_server.rs

```diff
@@ -632,8 +504,8 @@ mod test {
             range_request_with_future_rev.revision, current_revision
         ))
         .to_string();
-        let message = KvServer::<DB>::check_range_request(
-            &range_request_with_future_rev,
+        let message = KvServer::<DB>::check_revision(
+            &UniRequest::RequestRange(range_request_with_future_rev),
```

Comment:
We don't need to match the error message. We can match the `tonic::Status::Code` directly